[2025-05-23T13:20:01.721-0600] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-23T13:20:01.757-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: libertyPr_health_score.send_to_elasticsearch scheduled__2025-05-22T00:00:00+00:00 [queued]>
[2025-05-23T13:20:01.772-0600] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: libertyPr_health_score.send_to_elasticsearch scheduled__2025-05-22T00:00:00+00:00 [queued]>
[2025-05-23T13:20:01.773-0600] {taskinstance.py:2867} INFO - Starting attempt 7 of 8
[2025-05-23T13:20:01.790-0600] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): send_to_elasticsearch> on 2025-05-22 00:00:00+00:00
[2025-05-23T13:20:01.806-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2162) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-23T13:20:01.810-0600] {standard_task_runner.py:72} INFO - Started process 2164 to run task
[2025-05-23T13:20:01.815-0600] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'libertyPr_health_score', 'send_to_elasticsearch', 'scheduled__2025-05-22T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/health_score_lpr.py', '--cfg-path', '/tmp/tmpcybf29ma']
[2025-05-23T13:20:01.821-0600] {standard_task_runner.py:105} INFO - Job 35: Subtask send_to_elasticsearch
[2025-05-23T13:20:01.914-0600] {task_command.py:467} INFO - Running <TaskInstance: libertyPr_health_score.send_to_elasticsearch scheduled__2025-05-22T00:00:00+00:00 [running]> on host 439130e6349d
[2025-05-23T13:20:02.050-0600] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='libertyPr_health_score' AIRFLOW_CTX_TASK_ID='send_to_elasticsearch' AIRFLOW_CTX_EXECUTION_DATE='2025-05-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='7' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-22T00:00:00+00:00'
[2025-05-23T13:20:02.051-0600] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-23T13:20:02.101-0600] {logging_mixin.py:190} INFO - ðŸ“‹ Columns in ranking file: ['FIBER_NODE', 'CMTS_ID', 'Critical', 'Good', 'Offline', 'Warning', 'Total_Modems', 'Critical_Rate', 'Warning_Rate', 'Node_Size_Amplified', 'Health_Score', 'Attention_Priority']
[2025-05-23T13:20:04.063-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/elasticsearch/_sync/client/__init__.py:403: SecurityWarning: Connecting to 'https://10.212.56.57:9200' using TLS with verify_certs=False is insecure
  _transport = transport_class(

[2025-05-23T13:20:04.564-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T13:20:05.069-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.514s]
[2025-05-23T13:20:05.091-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T13:20:05.139-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.049s]
[2025-05-23T13:20:05.158-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T13:20:05.216-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.058s]
[2025-05-23T13:20:05.234-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T13:20:05.284-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.050s]
[2025-05-23T13:20:05.303-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T13:20:05.348-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.046s]
[2025-05-23T13:20:05.374-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T13:20:05.434-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.060s]
[2025-05-23T13:20:05.469-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T13:20:05.542-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.074s]
[2025-05-23T13:20:05.551-0600] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1097: InsecureRequestWarning: Unverified HTTPS request is being made to host '10.212.56.57'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2025-05-23T13:20:05.570-0600] {_transport.py:349} INFO - PUT https://10.212.56.57:9200/_bulk [status:200 duration:0.019s]
[2025-05-23T13:20:05.571-0600] {logging_mixin.py:190} INFO - âœ… Sent 3603 documents to Elasticsearch index 'hs_lpr'
[2025-05-23T13:20:05.573-0600] {python.py:240} INFO - Done. Returned value was: None
[2025-05-23T13:20:05.591-0600] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-23T13:20:05.592-0600] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=libertyPr_health_score, task_id=send_to_elasticsearch, run_id=scheduled__2025-05-22T00:00:00+00:00, execution_date=20250522T000000, start_date=20250523T192001, end_date=20250523T192005
[2025-05-23T13:20:05.683-0600] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-23T13:20:05.748-0600] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-23T13:20:05.752-0600] {local_task_job_runner.py:245} INFO - ::endgroup::
